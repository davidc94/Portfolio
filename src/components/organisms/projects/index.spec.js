import React from 'react';
import renderer from 'react-test-renderer';
import Projects from './index';

describe('Projects component', () => {
  it('render Projects as expected', () => {
    const render = renderer.create(
      <Projects />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});