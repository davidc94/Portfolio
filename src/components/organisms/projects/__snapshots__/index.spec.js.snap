// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component render Projects as expected 1`] = `
<div
  className="sc-bwzfXH bejfXd"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
  width="full"
>
  <div
    className="sc-bwzfXH bejfXd"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
    width="full"
  >
    <div
      className="sc-bwzfXH iewMrH"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-bwzfXH iewMrH"
      style={
        Object {
          "flex": 20,
        }
      }
    >
      <p
        className="sc-bdVaJa foxOvY"
        color="#A8DADC"
        size="large"
      >
        Projects
      </p>
    </div>
  </div>
  <div
    className="sc-bwzfXH bejfXd"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
    width="full"
  >
    <div
      className="sc-bwzfXH iewMrH"
      style={
        Object {
          "flex": 1,
        }
      }
    />
    <div
      className="sc-bwzfXH iewMrH"
      style={
        Object {
          "flex": 200,
          "flexDirection": "column",
        }
      }
    >
      <p
        className="sc-bdVaJa fSWxyT"
        color="#EEE5E9"
        size="base"
      >
        Here's a list of stuff that i've done in my life that I'm pretty proud of üëè
      </p>
      <div
        className="sc-bwzfXH hycTDa"
      >
        <li
          className="sc-htpNat jDCxMu"
          color="#EEE5E9"
          size="base"
        >
          This website
        </li>
        <li
          className="sc-htpNat jDCxMu"
          color="#EEE5E9"
          size="base"
        >
          Smart Home App (Uni dissertation)
        </li>
      </div>
    </div>
  </div>
</div>
`;
